core/shared/src/main/scala/org/http4s/headers/Sunset.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2013 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s
+package headers
+
+import cats.parse.Parser
+import org.typelevel.ci._
+
+object Sunset {
```

Comment:
Could we add a bit of ScalaDoc with a link to the RFC? https://www.rfc-editor.org/rfc/rfc8594 I was not aware a couple minutes ago if this was a standard or customary header.